### PR TITLE
Create 2021-02-13-Cream-Finance.md

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

**Comment by kishoian** [comment]
Help me to understand, please. What events in the timeline are considered significant? Is what I wrote enough or do I need more details? I can specify each translation in the smart contract, but then it will be too much, but if you do not write about translations, then there are too few events. Every hack is the same: the attack started, the company announced the hack, and the attack ended

**Comment by kishoian** [comment]
Does the Bitstamp article have a good timeline? I think not. But if so, then I can write such with links to statements

File: content/attacks/posts/2021-02-13-Cream-Finance.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Alpha Homora V2 Exploiter: [0x905315602ed9a854e325f692ff82f58799beab57](https://etherscan.io/address/0x905315602ed9a854e325f692ff82f58799beab57)
+
+## Losses
+
+Approximately $37.5 million:
+
+- ~4,263,138 DAI
+- ~4,424,580 USDC
+- ~5,647,242 USDT
+- ~13,244 WETH
```
Comment (JediFaust) [inline content/attacks/posts/2021-02-13-Cream-Finance.md:27]: Add worth of WETH at the moment of an incident, It can be calculated easily.

**Review by JediFaust — CHANGES_REQUESTED** [review]
It's unclear which protocol suffered from the hack, and how CREAM and Alpha Homora related to each other. Please, extend an article.

**Comment by kishoian** [comment]
I see that you've changed the categories on the website once again. You asked for feedback, so here are my thoughts:

I believe that categories should be distinctly segregated and should not overlap, as it can lead to confusion. We could classify all the articles based on the type of attacks; I have sketched out a rough idea below.

Main Categories:

- Protocol Exploits
Description: Articles about hacks that occur due to vulnerabilities in the protocols themselves.

- Interaction Attacks
Description: Articles about attacks based on manipulating interactions between smart contracts or protocols, including reentrancy, flash loans, etc.

- Key and Access Thefts
Description: Articles about cases of private key theft, phishing, and other methods of unauthorized access.

- Data Manipulation
Description: Articles about oracle attacks, price manipulations, and other ways of data tampering.

As for the current elements that are designated as categories, these could be integrated as tags. Often, a hack simultaneously falls under multiple descriptors such as bridge hack, DeFi, and protocol vulnerability. Labeling these as categories might be unconventional, but employing them as tags would be more appropriate.

Only tags without categories be better also

**Comment by JediFaust** [comment]
> I see that you've changed the categories on the website once again. You asked for feedback, so here are my thoughts:
> 
> I believe that categories should be distinctly segregated and should not overlap, as it can lead to confusion. We could classify all the articles based on the type of attacks; I have sketched out a rough idea below.
> 
> Main Categories:
> 
>     * Protocol Exploits
>       Description: Articles about hacks that occur due to vulnerabilities in the protocols themselves.
> 
>     * Interaction Attacks
>       Description: Articles about attacks based on manipulating interactions between smart contracts or protocols, including reentrancy, flash loans, etc.
> 
>     * Key and Access Thefts
>       Description: Articles about cases of private key theft, phishing, and other methods of unauthorized access.
> 
>     * Data Manipulation
>       Description: Articles about oracle attacks, price manipulations, and other ways of data tampering.
> 
> 
> As for the current elements that are designated as categories, these could be integrated as tags. Often, a hack simultaneously falls under multiple descriptors such as bridge hack, DeFi, and protocol vulnerability. Labeling these as categories might be unconventional, but employing them as tags would be more appropriate.
> 
> Only tags without categories be better also

Appreciate sharing thoughts! Although, I think more accurate sepparation 

> I see that you've changed the categories on the website once again. You asked for feedback, so here are my thoughts:
> 
> I believe that categories should be distinctly segregated and should not overlap, as it can lead to confusion. We could classify all the articles based on the type of attacks; I have sketched out a rough idea below.
> 
> Main Categories:
> 
>     * Protocol Exploits
>       Description: Articles about hacks that occur due to vulnerabilities in the protocols themselves.
> 
>     * Interaction Attacks
>       Description: Articles about attacks based on manipulating interactions between smart contracts or protocols, including reentrancy, flash loans, etc.
> 
>     * Key and Access Thefts
>       Description: Articles about cases of private key theft, phishing, and other methods of unauthorized access.
> 
>     * Data Manipulation
>       Description: Articles about oracle attacks, price manipulations, and other ways of data tampering.
> 
> 
> As for the current elements that are designated as categories, these could be integrated as tags. Often, a hack simultaneously falls under multiple descriptors such as bridge hack, DeFi, and protocol vulnerability. Labeling these as categories might be unconventional, but employing them as tags would be more appropriate.
> 
> Only tags without categories be better also

Thanks for sharing thoughts! I think this categories does not include 51% attack and front-run attacks for example. And the main categories should include wide range of attacks as well. I think, this list is pretty solid, but need to be reworked and extended.

**Comment by JediFaust** [comment]
Article is on pause, since there is another, earlier article is on pending state

**Comment by kishoian** [comment]
oh, I didn't notice it. *crying

**Comment by JediFaust** [comment]
No worries) It took time to figure it out, since articles was named differently

**Comment by JediFaust** [comment]
The same incident was already merged, closing

**Pull request closed without merging**